.github/workflows/test_and_release.yml

```diff
@@ -34,14 +34,14 @@ jobs:
             name: "Linux",
             os: ubuntu-latest
           }
-        - {
-            name: "MacOSX",
-            os: macos-latest
-          }
-        - {
-            name: "Windows",
-            os: windows-latest
-          }
+        # - {
+        #     name: "MacOSX",
+        #     os: macos-latest
+        #   }
```

Comment:
We could enable Mac pretty easily -- It doesn't require any addition dependencies and the plain VTK wheel should work fine. We would just need to break the `vtk-osmesa` install to only run on linux.